pkgs/development/python-modules/experiment-utilities/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, buildPythonPackage
+, cloudpickle
+, dill
+, fasteners
+, fetchFromGitLab
+, qgrid
+, ipywidgets
+, odfpy
+, scipy
+, plotly
+, pytestCheckHook
+, pyyaml
+, tabulate
+, tensorboard
+, torch
+}:
+
+buildPythonPackage {
+  pname = "experiment-utilities";
+  version = "0.3.4";
+
+  src = fetchFromGitLab {
+    owner = "creinke";
+    repo = "exputils";
+    domain = "gitlab.inria.fr";
+    # rev = "refs/tags/v${version}";
+    rev = "164219c11385605501482f8c72832e20844331c4";
```

Comment:
Whats up with this?

Reply:
Actually, there is neither tags nor releases on the Gitlab repo.
This hash corresponds to the commits which acts as the tag for this release.
I will try to get the author tagging the latest release.
Otherwise, either we use `fetchPypi` or we stick with this method.

Reply:
There is now a tag available.